gui/default/index.html

```diff
@@ -385,22 +385,22 @@ <h4 class="panel-title">
                   <table class="table table-condensed table-striped table-auto">
                     <tbody>
                       <tr ng-show="folder.label != undefined && folder.label.length > 0">
-                        <th><span class="fas fa-fw fa-info-circle"></span>&nbsp;<span translate>Folder ID</span></th>
-                        <td class="text-right no-overflow-ellipse">{{folder.id}}</td>
+                        <th class="col-fixed"><span class="fas fa-fw fa-info-circle"></span>&nbsp;<span translate>Folder ID</span></th>
+                        <td class="col-flex text-right no-overflow-ellipse">{{folder.id}}</td>
```

Comment:
It's related in that we want to use the same table cell styles here and in the fancytree table, but don't want the style tied to the cell type nor the order of the cells. So the styles in overrides.css are refactored to use an explicit class name "col-fixed" instead of the element type "th".

It could be a separate PR, but it would be a strange one because the refactor has no effect on its own. I will investigate a different approach to this CSS, with the previous behavior as default and the option to specify a class and override it.